Sources/XMLCoder/Box/UnsignedIntegerBox.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UnsignedIntegerBox.swift
+//  XMLCoder
+//
+//  Created by Vincent Esche on 12/17/18.
+//
+
+import Foundation
+
+internal struct UnsignedIntegerBox: Equatable {
+    typealias Unboxed = Int
+    
+    let unboxed: Unboxed
+    
+    init<Integer: UnsignedInteger>(_ unboxed: Integer) {
+        self.unboxed = Unboxed(unboxed)
```

Comment:
hm, does this mean that we'll truncate `Int64` on platforms where `Int` is actually `Int64`, like Apple Watch?

Reply:
I changed it to using `Int64` and `UInt64` instead of `Int` and `UInt` as 
`Unbounded`. This should work with AppleWatch, right?

Fixed.